garage/envs/normalized_env.py

```diff
@@ -86,7 +87,7 @@ def __setstate__(self, d):
 
     @overrides
     def step(self, action):
-        if isinstance(self.action_space, gym.spaces.Box):
+        if isinstance(self.action_space, Box):
```

Comment:
We explicitly refer to `gym.spaces.Box` for a reason: to make it clear that we mean the Gym version of box and not the Garage version of box. Please keep this (and all others) `gym.spaces.Foo`